homepage/homepage/app/docs/[...slug]/schemas/accounts.mdx

```diff
@@ -0,0 +1,70 @@
+import { CodeGroup } from "@/components/forMdx";
+
+# Accounts & migrations
+
+Accounts are a fundamental building block for user identity and data organization in Jazz applications, providing a structured way to manage both public and private user data while maintaining type-safety and access control.
+
+Account data and metadata are publicly visible to other users.
+
+For private data, we suggest nesting privately owned CoValues within the account.
+
+## Accounts
+
+To create a custom account type, extend the `Account` class and register it with the Jazz runtime.
+
+<CodeGroup>
+```ts
+import { Account, CoMap } from "jazz-tools"; 
+
+class MyAccountRoot extends CoMap {
+  privateString = co.string; // Readable by the account owner
+}
+
+class MyAccount extends Account {
+  root = co.ref(MyAccountRoot);
+  publicString = co.string; // Readable by everyone
+}
+```
+</CodeGroup>
+
+### Schema
+
+When passing the account class to the Jazz runtime, the schema is used to type account data.
+
+<CodeGroup>
+```ts
+// React
+const Jazz = createJazzReactApp({
+  AccountSchema: MyAccount,
+});
+
+// Vue
+const Jazz = createJazzVueApp({
+  AccountSchema: MyAccount,
+});
+```
```

Comment:
Also add react native here?

And put in a separate CodeGroup so that they can be copied separately with the copy button

Not related to this PR:
We'll have a lot of instances like this where the code is different for each framework.. We can do a global toggle maybe

Reply:
Will put React Native example in, and separate CodeGroups 

Yeah a language-toggleable group would be a good component to make.